components/list/list-item.js

```diff
@@ -114,8 +30,7 @@ class ListItem extends RtlMixin(LitElement) {
 				padding: 0.55rem 0;
```

Comment:
The elements can still style the what is rendered in the mixin. When inheriting from the mixin you need to know everything about how it works. I wouldn't want people making new `list-items` all the time. I never want to see a new `list-item` in HMC repo. So this isn't so bad if we consider new `list-items` patterns should be rare and far between.